.github/workflows/test-suite-desktop-e2e.yml

```
@@ -96,15 +134,15 @@ jobs:
       - name: Extract Trezor-user-env and Regtest logs
-        if: ${{ ! cancelled() }}
+        if: ${{ ! cancelled() }} && env.SKIP_TESTS != 'true'
         run: |
           docker cp docker-trezor-user-env-unix-1:/trezor-user-env/logs/debugging.log trezor-user-env-debugging.log || true
           docker cp docker-trezor-user-env-unix-1:/trezor-user-env/logs/emulator_bridge.log tenv-emulator-bridge-debugging.log || true
           docker cp docker-trezor-user-env-unix-1:/trezor-user-env/docker/version.txt trezor-user-env-version.txt || true
           docker logs docker-electrum-regtest-1 > electrum-regtest.txt || true
 
       - name: Upload Trezor-user-env and Regtest logs
-        if: ${{ ! cancelled() }}
+        if: ${{ ! cancelled() }}  && env.SKIP_TESTS != 'true'
```

actionlint (1.7.4)

if: condition "${{ ! cancelled() }}  && env.SKIP_TESTS != 'true'" is always evaluated to true because extra characters are around ${{ }}

(if-cond)